fe1-web/src/features/evoting/reducer/ElectionKeyWatcher.ts

```diff
@@ -0,0 +1,21 @@
+import { Store } from 'redux';
+
+import { PublicKey } from 'core/objects';
+
+import { getElectionKeyByElectionId } from './ElectionKeyReducer';
+
+/**
+ * Watches the redux store for new message#witness messages for lao#greet messages since they only
+ * become valid after they are witnessed by the corresponding frontend
+ * @remark Implemented analogous to 'makeMessageStoreWatcher'
+ * @param store The redux store to watch
+ * @param laoGreetSignatureHandler The function to call when a signature is added to a lao#greet message
+ */
```

Comment:
This comment is wrong here😅 I fixed in in #968:

https://github.com/dedis/popstellar/blob/work-fe1-nico-secret-ballot-election-ui/fe1-web/src/features/evoting/reducer/ElectionKeyWatcher.ts#L7-L12